### ISSUE TITLE
problem when launching this great module

### ISSUE THREAD

**Issue by bcharlier** [HMMA238-2020/git-tutorial#1]
Hi there,

thank you for developing this great tool. However, I get a strange error when using it:
```
SyntaxError: EOL while scanning string literal
```

**Comment by lvl0-statistician** [comment]
hello, 

after some code inspection, the error seems to come from a missing ` ' ` in the `plt` function.
i am working on it and will submit a PR when it's fixed.

_edit : error fixed and PR submitted_

**Comment by AmelieVernay** [comment]
I agree with lvl0-statistician and also mention that the indentation level was wrong on some lines (though this was not the reason for the `SyntaxError: EOL while scanning string literal` error)
